src/util/DataTransformationUtils.ts

```diff
@@ -18,7 +18,7 @@ export class DataTransformationUtils {
      */
```

Comment:
Hi @eduardoweiland just changed the title of the PR to match the issue's tittle.
Thanks for your feedback. Cheers!